From 4b8a12785c96c33a77eb59fdd1c088d25978f7d8 Mon Sep 17 00:00:00 2001
From: Eric Garver <e@erig.me>
Date: Wed, 26 Jul 2017 10:10:19 -0400
Subject: [PATCH] Add NFSv3 service.

This is distinct from the NFS service (v4) because it also opens up UDP
ports.

Fixes: RHBZ#1462088
(cherry picked from commit a127d697177b78b7f9b766deb978efd95590a2ac)
---
 config/Makefile.am       | 1 +
 config/services/nfs3.xml | 7 +++++++
 2 files changed, 8 insertions(+)
 create mode 100644 config/services/nfs3.xml

diff --git a/config/Makefile.am b/config/Makefile.am
index bdc5651c154c..1035c9f940a9 100644
--- a/config/Makefile.am
+++ b/config/Makefile.am
@@ -173,6 +173,7 @@ CONFIG_FILES = \
 	services/ms-wbt.xml \
 	services/mysql.xml \
 	services/nfs.xml \
+	services/nfs3.xml \
 	services/nrpe.xml \
 	services/ntp.xml \
 	services/openvpn.xml \
diff --git a/config/services/nfs3.xml b/config/services/nfs3.xml
new file mode 100644
index 000000000000..4075d48211bd
--- /dev/null
+++ b/config/services/nfs3.xml
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>NFS3</short>
+  <description>The NFS3 protocol is used to share files. You will need to have the NFS tools installed and properly configure your NFS server for this option to be useful.</description>
+  <port protocol="tcp" port="2049"/>
+  <port protocol="udp" port="2049"/>
+</service>
-- 
2.12.0

